packages/graphql/src/schema/make-augmented-schema.ts

```diff
@@ -751,6 +741,8 @@ function makeAugmentedSchema({
         }, {}),
     };
 
+    composer.removeEmptyTypes(composer.get("Mutation") as ObjectTypeComposer, new Set());
```

Comment:
Honestly confused by this - could you get it working in the end or no? I just don't see the other lines we discussed in this diff.

Reply:
Nah it shouldn't be in here good spot. I was when I was experimenting with a fix.